llama_index/finetuning/embeddings/common.py

```diff
@@ -78,7 +77,16 @@ def generate_qa_embedding_pairs(
         for node in nodes
     }
 
-    llm = llm or OpenAI(model="gpt-3.5-turbo")
```

Comment:
`resolve_llm` no longer automatically falls back

I think a better fix is `llm = llm or resolve_llm("default")`

Reply:
It will print an appropriate warning no?

Reply:
I don't think that would help, because if LLM is None, the resolve_llm function will be called and in that 
![Screenshot 2023-11-15 at 10 36 06 PM](https://github.com/run-llama/llama_index/assets/19616698/3dec100e-9714-470e-b528-bf3ffd486b83)
mock_llm will be set which will only generate the string with "text".

![Uploading Screenshot 2023-11-15 at 10.36.47 PM.png…]()

Reply:
Yea, that's why I would use `llm = llm or resolve_llm("default")` instead of passing in anything

Reply:
actually one sec, lemme push my idea :)

Reply:
adding llm = resolve_llm(llm) will circle back to open ai function (same issue which I faced). I was using Azure Open AI and it threw authentication error, and only way to diagnose was to look up the code.

Reply:
OK sure. So final fix, don't make llm an optional argument in the function def

Reply:
Agree with you, removed it

Reply:
there was a typo so changed it, in commit instead of typo, by mistake I wrote type. Sorry for that